scripts/update.sh

```diff
@@ -37,6 +37,12 @@ fi
 
 . "$REQUIRE_FILE"
 
+# Check for wallabag .git folder
+if [ ! -d .git ]; then
+    echo "Can not update because wallabag wasn't installed using git (see https://doc.wallabag.org/en/admin/upgrade.html#upgrade-on-a-shared-hosting). Aborting.";
+    exit 1;
```

Comment:
Special exit code maybe ?

Reply:
Like 2 instead of 1?

Reply:
For instance, just so we know where the issue comes when using an automatic system.

Reply:
Okay